File: extract_text.js
const fs = require('fs');
const content = fs.readFileSync('prevail_text_utf8.txt', 'utf-8');
const strings = content.match(/(["'])(?:(?=(\\?))\2.)*?\1/g) || [];
const validStrings = strings.filter(s => {
  const stripped = s.slice(1, -1);
  return stripped.length > 30 && /^[A-Z][a-z0-9 ,\.-]+$/.test(stripped) && stripped.split(' ').length > 4;
});
console.log(Array.from(new Set(validStrings)).join('\n'));

File: extract_text.py
import re
with open('prevail_text_utf8.txt', 'r', encoding='utf-8') as f:
    content = f.read()
    
# Extract strings
strings = re.findall(r'(["\'])(.*?)\1', content)
for _, text in strings:
    text = text.strip()
    if len(text) > 40 and ' ' in text and not re.search(r'[{}<>\\]', text):
        if re.search(r'[a-z]{3}', text):
            print(text)
